src/toil/utils/toilStatus.py

```diff
@@ -205,8 +205,6 @@ def getStatus(jobStoreName):
             jobstore = Toil.resumeJobStore(jobStoreName)
         except NoSuchJobStoreException:
             return 'QUEUED'
-        except NoSuchFileException:
```

Comment:
This should probably be added back, especially now that it's no longer being called.